src/panel-manager.js

```diff
@@ -9,6 +9,14 @@ genotet.panelManager = {};
 genotet.panelManager.COLLAPSED_WIDTH = 32;
 /** @const {number} */
 genotet.panelManager.TRANSITION_TIME = 300;
+/** @const {number} */
+genotet.panelManager.TOGGLE_BTN_HEIGHT = 50;
+/** @const {number} */
+genotet.panelManager.TAB_MARGIN_BOTTOM_DIFFERENCE = 20;
```

Comment:
It represents the different value between the width and margin-bottom of the tabs container. Because the margin-bottom value has to be adjusted with the width value.